### PR TITLE
update webhook instructions

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees @spadgett PTAL

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -235,9 +235,12 @@
 <ol>
   <li>From the Web Console homepage, navigate to your project</li>
   <li>Click on Browse &gt; Builds</li>
-  <li>From the view for your Build click on the button to copy your GitHub webhook</li>
-  <li>Navigate to your repository on GitHub and click on repository settings &gt; webhooks</li>
-  <li>Paste your webhook URL provided by OpenShift &mdash; that's it!</li>
+  <li>Click the nodejs-example link that represents the BuildConfig for this example application</li>
```
Comment (spadgett) [inline views/index.html]: The actual build config name might not be nodejs-example depending on how you created the app.

Comment (gabemontero) [inline views/index.html]: yep ... rework to that line pushed

File: views/index.html [hunk 2]
```diff
@@ -235,9 +235,12 @@
...
   <li>Click on Browse &gt; Builds</li>
-  <li>From the view for your Build click on the button to copy your GitHub webhook</li>
-  <li>Navigate to your repository on GitHub and click on repository settings &gt; webhooks</li>
-  <li>Paste your webhook URL provided by OpenShift &mdash; that's it!</li>
+  <li>Click the link with your BuildConfig name</li>
+  <li>Click the Configuration tab</li>
+  <li>Click the "Copy to clipboard" icon to the right of the "GitHub webhook URL" field</li>
+  <li>Navigate to your repository on GitHub and click on repository settings &gt; webhooks &gt; Add webhook</li>
+  <li>Paste your webhook URL provided by OpenShift</li>
+  <li>Leave the content type as "application/json" and adjust the secret, SSL verifictation, and other Webhook parameters as necessary for your specific environment</li>
```
Comment (spadgett) [inline views/index.html]: If the defaults work with OpenShift, we might just say that you can take the defaults (unless there is a value they really should be changing).

Comment (bparees) [inline views/index.html]: +1

also verification has a typo in it.

Comment (gabemontero) [inline views/index.html]: update pushed based on last set of comments

**Comment by bparees** [comment]
lgtm

**Pull request merged**